static/scripts/audit-report/audit.ts

```diff
@@ -32,6 +31,12 @@ import {
 } from "./types";
 import { getTxInfo } from "./utils/getTransaction";
 
+if (!process.env.SUPABASE_URL || !process.env.SUPABASE_KEY) {
+  throw new Error("Missing Supabase env variables.");
+}
+
+const supabase = createClient(process.env.SUPABASE_URL, process.env.SUPABASE_KEY);
```

Comment:
@pavlovcik What value did you set in the `process.env.SUPABASE_KEY`? It should be a [public anon key](https://supabase.com/docs/guides/api/api-keys#the-anon-key), not a [service key](https://supabase.com/docs/guides/api/api-keys#the-servicerole-key) since this value can be read from the frontend. Just a double check.

Reply:
@rndquu Would you like me to rename it to avoid confusion?

Reply:
> @pavlovcik What value did you set in the `process.env.SUPABASE_KEY`? It should be a [public anon key](https://supabase.com/docs/guides/api/api-keys#the-anon-key), not a [service key](https://supabase.com/docs/guides/api/api-keys#the-servicerole-key) since this value can be read from the frontend. Just a double check.

I tried looking from my phone but the Supabase UI is unusable and I couldn't confirm. I'm pretty sure I used the anon key but I can re set it when I get back to my computer later.

Reply:
> @rndquu Would you like me to rename it to avoid confusion?

Yeah, it could be more explicit, smth like `SUPABASE_PUBLIC_ANON_KEY`

Reply:
I just created `SUPABASE_ANON_KEY` 

I deleted `SUPABASE_KEY`

@ubiquity/software-development

Reply:
@pavlovcik I renamed them in the code as well now